inst/tests/tests.Rraw

```diff
@@ -18330,3 +18330,51 @@ if (test_bit64) {
   apple = data.table(id = c("a", "b", "b"), time = c(1L, 1L, 2L), y = i64v[1:3])
   test(2248, dcast(apple, id ~ time, value.var = "y"), data.table(id = c('a', 'b'), `1` = i64v[1:2], `2` = i64v[4:3], key='id'))
 }
+
+# 5885 implement frev
+d = c(NA, NaN, Inf, -Inf)
+test(2249.00, frev(c(FALSE, NA),    copy=TRUE), c(NA, FALSE))
+test(2249.01, frev(c(0L, NA),       copy=TRUE), c(NA, 0L))
+test(2249.02, frev(d,               copy=TRUE), c(-Inf, Inf, NaN, NA))
+test(2249.03, frev(c(NA, 1, 0+2i),  copy=TRUE), c(0+2i, 1, NA))
+test(2249.04, frev(as.raw(0:1),     copy=TRUE), as.raw(1:0))
+test(2249.05, frev(NULL,            copy=TRUE), NULL)
+test(2249.06, frev(character(5),    copy=TRUE), character(5))
+test(2249.07, frev(integer(0),      copy=TRUE), integer(0))
+test(2249.08, frev(list(1, "a"),    copy=TRUE), list("a", 1))
+test(2249.09, frev(c(0L, NA),       copy=FALSE), c(NA, 0L))
+test(2249.10, frev(d,               copy=FALSE), c(-Inf, Inf, NaN, NA))
+test(2249.11, frev(c(NA, 1, 0+2i),  copy=FALSE), c(0+2i, 1, NA))
+test(2249.12, frev(as.raw(0:1),     copy=FALSE), as.raw(1:0))
+test(2249.13, frev(NULL,            copy=FALSE), NULL)
+test(2249.14, frev(character(5),    copy=FALSE), character(5))
+test(2249.15, frev(integer(0),      copy=FALSE), integer(0))
+test(2249.16, frev(list(1, "a"),    copy=FALSE), list("a", 1))
+test(2249.17, frev(1:1e2,           copy=TRUE),  rev(1:1e2))
+# copy arguments
+x = 1:3
+test(2249.21, {frev(x, copy=TRUE);  x}, 1:3)
+test(2249.22, {frev(x, copy=FALSE); x}, 3:1)
```

Comment:
This is OK, but shouldn't we also have a test showing `frev()` output and `x` have the same address?

Reply:
Should we also add a test like `frev(x[1:2], copy=FALSE); x` to make sure the original `x` is not altered when `frev()` is run on a subset? This is more like a test of base R behavior, but would be good to be cautious.